miraclelinux/meta-debian-extended
# Add bsd-mailx, smartmontools packages.

# Purpose of pull request

bsd-mailx 
smartmontools

This pr adds smartmontoolspackage based on meta-openembedded's meta-oe/recipes-extended/smartmontools/smartmontools_7.0.bb
bsd-mailx is a dependency of smartmontools.

# Test
## How to test

ref. https://wiki.archlinux.jp/index.php/S.M.A.R.T.
According to the above URL, SMART-compatible storage such as HDD is required to check the operation.
In this case, since we are using QEMU environment, we will only check the command execution.

Checking commands
Execute smartctl, smartd and update-smart-drivedb.

## Test result

```
oot@qemuarm64:~# smartctl -h
smartctl 6.6 2017-11-05 r4594 [aarch64-linux-4.19.205-cip56] (local build)
Copyright (C) 2002-17, Bruce Allen, Christian Franke, www.smartmontools.org

Usage: smartctl [options] device

============================================ SHOW INFORMATION OPTIONS =====

  -h, --help, --usage
         Display this help and exit

  -V, --version, --copyright, --license
         Print license, copyright, and version information and exit
... <snip> ...
root@qemuarm64:~# smartd -h
smartd 6.6 2017-11-05 r4594 [aarch64-linux-4.19.205-cip56] (local build)
Copyright (C) 2002-17, Bruce Allen, Christian Franke, www.smartmontools.org

Usage: smartd [options]

  -A PREFIX, --attributelog=PREFIX
        Log ATA attribute information to {PREFIX}MODEL-SERIAL.ata.csv

  -B [+]FILE, --drivedb=[+]FILE
        Read and replace [add] drive database from FILE
        [default is +/etc/smart_drivedb.h
         and then    /usr/share/smartmontools/drivedb.h]

  -c NAME|-, --configfile=NAME|-
        Read configuration file NAME or stdin
        [default is /etc/smartd.conf]
... <snip> ...
root@qemuarm64:~# update-smart-drivedb -h
smartmontools 6.6 drive database update script

Usage: /usr/sbin/update-smart-drivedb [OPTIONS] [DESTFILE]

  -s SMARTCTL     Use SMARTCTL for syntax check ('-s -' to disable)
                  [default: /usr/sbin/smartctl]
  -t TOOL         Use TOOL for download: curl wget lynx svn
                  [default: first one found in PATH]
... <snip> ...
```
```
root@qemuarm64:~# smartctl --scan
# scan_smart_devices: glob(3) aborted matching pattern /dev/discs/disc*
```
